modules/api/src/main/java/org/apache/ignite/tx/IgniteTransactions.java

```diff
@@ -257,26 +267,9 @@ default <T> T runInTransaction(Function<Transaction, T> clo) throws TransactionE
      * @throws TransactionException If a transaction can't be finished successfully.
      */
     default <T> T runInTransaction(Function<Transaction, T> clo, @Nullable TransactionOptions options) throws TransactionException {
-        Objects.requireNonNull(clo);
-
-        Transaction tx = begin(options);
-
-        try {
-            T ret = clo.apply(tx);
-
-            tx.commit();
-
-            return ret;
-        } catch (Throwable t) {
-            // TODO FIXME https://issues.apache.org/jira/browse/IGNITE-17838 Implement auto retries
-            try {
-                tx.rollback(); // Try rolling back on user exception.
-            } catch (Exception e) {
-                t.addSuppressed(e);
-            }
-
-            throw t;
-        }
+        long startTimestamp = System.currentTimeMillis();
```

Comment:
Please add a comment, why it's valid to have non-unique startTimestamp here in contrast to common transactions beginTimestamp 
`HybridTimestamp beginTimestamp = clockService.now(); // Tick to generate new unique id.`